src/sinks/azure_monitor_logs.rs

```diff
@@ -99,6 +103,17 @@ pub struct AzureMonitorLogsConfig {
     #[serde(default)]
     pub request: TowerRequestConfig,
 
+    /// Use this option to customize the log field used as [`TimeGenerated`][1] in Azure.
+    ///
+    /// The setting of `log_schema.timestamp_key`, usually `timestamp`, is used here by default.
+    /// This field should be used in rare cases where `TimeGenerated` should point to specific log
+    /// field. For example, use this field to set the log field `source_timestamp` as holding the
```

Comment:
```suggestion
    /// This field should be used in rare cases where `TimeGenerated` should point to a specific log
    /// field. For example, use this field to set the log field `source_timestamp` as holding the
```

Reply:
Fixed in [e7ce8d5](https://github.com/vectordotdev/vector/pull/16475/commits/e7ce8d5715130860abee91b0e1fbfe1104b7ac0a)